script.js

```diff
@@ -0,0 +1,58 @@
+function selectop() {
+    var operval = document.getElementById("opervalue").value
+    switch (operval) {
+        case '+':
+            sum();
+            break;
+        case "-":
+            subtract();
+            break;
+        case "*":
+            multiply();
+            break;
+        case "/":
+            divide();
+            break;
+        default:
+            badop()
+            break;
+    }
+}
+
+function sum() {
+    var firstval = parseInt(document.getElementById("firstvalue").value)
+    var secondval = parseInt(document.getElementById("secondvalue").value)
+    var answer = firstval + secondval
+    var outputElement = document.getElementById("output")
+    outputElement.innerHTML = answer
+}
+
+function subtract() {
+    var firstval = parseInt(document.getElementById("firstvalue").value)
+    var secondval = parseInt(document.getElementById("secondvalue").value)
+    var answer = firstval - secondval
+    var outputElement = document.getElementById("output")
+    outputElement.innerHTML = answer
+}
+
+function multiply() {
+    var firstval = parseInt(document.getElementById("firstvalue").value)
+    var secondval = parseInt(document.getElementById("secondvalue").value)
+    var answer = firstval * secondval
+    var outputElement = document.getElementById("output")
+    outputElement.innerHTML = answer
+}
+
+function divide() {
+    var firstval = parseInt(document.getElementById("firstvalue").value)
+    var secondval = parseInt(document.getElementById("secondvalue").value)
+    var answer = firstval / secondval
+    var outputElement = document.getElementById("output")
+    outputElement.innerHTML = answer
+}
```

Comment:
This is a lot of repetition for a one line difference. You'd want to DRY this up, it could all easily live in your one function.

```JavaScript
function selectop() {
    var operval = document.getElementById("opervalue").value
    var firstval = parseInt(document.getElementById("firstvalue").value)
    var secondval = parseInt(document.getElementById("secondvalue").value)
    var answer

    switch (operval) {
        case '+':
            answer = firstval + secondval
            break
        case '-':
            answer = firstval - secondval
            break
        case '*':
            answer = firstval * secondval
            break
        case '/':
            answer = firstval / secondval
            break
        default:
            answer = 'Please check your operator and try again'
            break
    }

    document.getElementById("output").innerHTML = answer
}
```